pkg/operator/ceph/cluster/mgr/config.go

```diff
@@ -59,7 +59,7 @@ func (c *Cluster) dashboardPort() int {
 func (c *Cluster) generateKeyring(m *mgrConfig) (string, error) {
 	user := fmt.Sprintf("mgr.%s", m.DaemonID)
 	/* TODO: the access string here does not match the access from the keyring template. should they match? */
```

Comment:
Should we address this too? I think it's about time :)

Reply:
Yeah, I wanted to ask. What is this about? 

Change
```
const (
	keyringTemplate = `
[mgr.%s]
	key = %s
	caps mon = "allow *"
	caps mds = "allow *"
	caps osd = "allow *"
`
)
```
to
```
const (
	keyringTemplate = `
[mgr.%s]
	key = %s
	caps mon = "allow profile mgr"
	caps mds = "allow *"
	caps osd = "allow *"
`
)
```
??

Reply:
Yes, would be nice to have a constant for each cap IMHO.

Reply:
Updated the template. That should suffice.

Reply:
What about using constants?

Reply:
You mean constant for `allow profile mgr` and `allow *` and reference it in the `keyringTemplate` (which is also a constant)?